pyfakefs/fake_filesystem_unittest.py

```diff
@@ -130,7 +130,7 @@ def copyRealFile(self, real_file_path, fake_file_path=None,
 
         Args:
           real_file_path: Path to the source file in the real file system.
-          fake_file_path: path to the destination file in the fake file system.
+          fake_file_path: path to the destination file in the fake file system.  Defaults to real_file_path.
```

Comment:
Sure:

Like this?

fake_file_path: path to the destination file in the fake file system.
            Defaults to real_file_path.

Or this?

fake_file_path: path to the destination file in the fake file system.
                         Defaults to real_file_path.

?